docs/content/preview/reference/configuration/yugabyted.md

```diff
@@ -87,6 +87,14 @@ For help with specific yugabyted commands, run 'yugabyted [ command ] -h'. For e
 $ ./bin/yugabyted start -h
 ```
 
+## Base directory
+
+By default, yugabyted uses the `/home/user/var` directory to store data, configurations, and logs.
```

Comment:
I think we should change this from `/home/user/var` to `$HOME/var`, as in the quick start docs.